File: store/src/main/java/fooddelivery/domain/CookStart.java
package fooddelivery.domain;

import fooddelivery.domain.*;
import fooddelivery.infra.AbstractEvent;
import java.util.*;
import lombok.*;

@Data
@ToString
public class CookStart extends AbstractEvent {

    private Long id;
    private String foodId;
    private String orderId;
    private String preference;
    private String status;

    public CookStart(OrderManage aggregate){
        super(aggregate);
    }
    public CookStart(){
        super();
    }
}

File: front/CookStart.py

from AbstractEvent import AbstractEvent
import json

class CookStart(AbstractEvent):
    id : int
    foodId : str
    orderId : str
    preference : str
    status : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.foodId = None
        self.orderId = None
        self.preference = None
        self.status = None

